dd-sdk-android-core/src/main/kotlin/com/datadog/android/api/storage/datastore/DataStoreHandler.kt

```diff
@@ -23,12 +23,14 @@ interface DataStoreHandler {
      * @param data to write.
      * @param version optional version for the entry.
      * If not specified will give the entry version 0 - even if that would be a downgrade from the previous version.
+     * @param callback to indicate whether the operation succeeded or not.
      * @param serializer to use to serialize the data.
      */
     fun <T : Any> setValue(
         key: String,
         data: T,
         version: Int = 0,
+        callback: DataStoreWriteCallback,
```

Comment:
For when we remove the expired datastore, because this is an async operation and therefore we need to know when it's finished in order to be sure that the initialization of the datastoremanager is complete.

Reply:
At the moment for `setValue` there is no specific use-case, but I felt that it makes sense to also have a callback here, since we have one for `removeValue`